### PR TITLE
Bug/343 - Fix reloading assets search list on adroid devices

### PR DESCRIPTION
**Pull request by jvrastil**

### CODE REVIEW

**Review by PitTxid — APPROVED** [review]

File: lib/logic/cubit/crypto_asset/crypto_asset_cubit.dart [hunk 1]
```diff
@@ -14,31 +14,31 @@ class CryptoAssetCubit extends Cubit<CryptoAssetState> {
...
-    emit(const CryptoAssetLoading());
+  Future<void> loadCryptoAsset(int pageKey) async {
+    emit(CryptoAssetLoading());
     try {
       final assets = await _marketplaceRepository.assetSearch(
         SearchAssetsRequest(
-          page: page,
-          pageSize: 100,
+          page: pageKey,
+          pageSize: Constants.pageSizeGames,
```
Comment (TomasPetrik) [inline lib/logic/cubit/crypto_asset/crypto_asset_cubit.dart]: Just fix this constant.

File: lib/view/widgets/discover_coins_content.dart [hunk 2]
```diff
@@ -29,7 +29,9 @@ class DiscoverCoinsContent extends StatefulWidget {
 class _DiscoverCoinsContentState extends State<DiscoverCoinsContent> {
   late CryptoAssetCubit cubit;
 
-  void addLoadEvent(int offset) => cubit.loadCryptoAsset(page: offset);
+  void addLoadEvent(int offset) {
```
Comment (TomasPetrik) [inline lib/view/widgets/discover_coins_content.dart:32]: You can still use fat arrow oparator if there is only one command inside method.

File: lib/logic/cubit/crypto_asset/crypto_asset_cubit.dart (hunk 1 above)
Comment (jvrastil) [reply lib/logic/cubit/crypto_asset/crypto_asset_cubit.dart]: done

File: lib/view/widgets/discover_coins_content.dart (hunk 2 above)
Comment (jvrastil) [reply lib/view/widgets/discover_coins_content.dart]: done

**Review by TomasPetrik — APPROVED** [review]

**Pull request merged**